parsing/ast_helper.mli

```diff
@@ -91,7 +91,7 @@ module Exp:
 
     val ident: ?loc:loc -> ?attrs:attrs -> lid -> expression
     val constant: ?loc:loc -> ?attrs:attrs -> constant -> expression
-    val let_: ?loc:loc -> ?attrs:attrs -> rec_flag -> value_binding list
+    val let_: ?loc:loc -> ?attrs:attrs -> rec_flag -> bool -> value_binding list
```

Comment:
This would be incoherent with the current style, where no labels are used for constructor arguments.  The best thing to do would probably be to introduce a let_bindings record, with the rec_flag, the lazy_flag and the value_binding list.  It would be used in Pexp_let, but also Pcl_let and Pstr_value (where 'lazy' would be rejected at type-checking time).